### PR TITLE
chore(gatsby-telemetry): Generate typings

### PR DESCRIPTION
**Pull request by ascorbic**
Add a typegen step to the gatsby-telemetry package, which is already fully converted to TS

### CODE REVIEW

File: packages/gatsby-telemetry/package.json [hunk 1]
```diff
@@ -50,9 +50,10 @@
   },
   "scripts": {
     "build": "babel src --out-dir lib --ignore \"**/__tests__\",\"**/__mocks__\" --extensions \".ts,.js\"",
-    "prepare": "cross-env NODE_ENV=production npm run build",
+    "prepare": "cross-env NODE_ENV=production npm run build && npm run typegen",
     "jest": "jest",
     "postinstall": "node src/postinstall.js || true",
+    "typegen": "rimraf \"lib/**/*.d.ts\" && tsc --emitDeclarationOnly --declaration --declarationDir lib/",
```
Comment (wardpeet) [inline packages/gatsby-telemetry/package.json:56]: Can you install rimraf as a devDep please

Comment (ascorbic) [reply packages/gatsby-telemetry/package.json:57]: Done

**Review by jamo — APPROVED** [review]
Nice!

**Review by jamo — APPROVED** [review]
👌

**Review by jamo — APPROVED** [review]

**Pull request merged**